Sources/SwiftDocC/Model/Rendering/DocumentationContentRenderer.swift

```diff
@@ -363,7 +363,17 @@ public class DocumentationContentRenderer {
         let isRequired = (node?.semantic as? Symbol)?.isRequired ?? false
 
         let estimatedTime = (node?.semantic as? Timed)?.durationMinutes.flatMap(formatEstimatedDuration(minutes:))
-
+        
+        // Add key information for property lists.
+        var propertyListKeyNames: TopicRenderReference.PropertyListKeyNames? = nil
+        if let nodePropertyListDetails = node?.symbol?.plistDetails  {
+            propertyListKeyNames = TopicRenderReference.PropertyListKeyNames(
+                titleStyle: (nodePropertyListDetails.customTitle != nil) ? .useDisplayName : .useRawKey,
+                rawKey: nodePropertyListDetails.rawKey,
+                displayName: nodePropertyListDetails.customTitle
+            )
+        }
```

Comment:
FYI: this pattern of 

```swift
var someValue: Something? = nil
if let somethingOptional {
   someValue = // create a `Something` based on the unwrapped `somethingOptional`
}
```

or

```swift
let someValue: Something?
if let somethingOptional {
   someValue = // create a `Something` based on the unwrapped `somethingOptional`
} else {
   someValue = nil 
}
```

Is good use of [`Optional.map(_:)`](https://developer.apple.com/documentation/swift/optional/map(_:)), which turns the pattern into 

```swift
let someValue = somethingOptional.map {
   // create a `Something` based on the unwrapped `$0`
}
```


```suggestion
        let propertyListKeyNames = node?.symbol?.plistDetails.map {
            TopicRenderReference.PropertyListKeyNames(
                titleStyle: ($0.customTitle != nil) ? .useDisplayName : .useRawKey,
                rawKey: $0.rawKey,
                displayName: $0.customTitle
            )
        }
```

Reply:
Resolved https://github.com/swiftlang/swift-docc/pull/1018/commits/d20f1ee1bfedc2d0765b9e6f22579d321b35ccb9